autoload/SpaceVim/custom.vim

```diff
@@ -209,9 +209,9 @@ function! s:load_glob_conf() abort
   if filereadable(global_dir . 'init.toml')
     let g:_spacevim_global_config_path = global_dir . 'init.toml'
     let local_conf = global_dir . 'init.toml'
-    let local_conf_cache = s:FILE.unify_path(expand(g:spacevim_data_dir.'/SpaceVim/conf/init.json'))
+    let local_conf_cache = s:FILE.unify_path(expand(g:spacevim_data_dir.'/SpaceVim/conf/' . fnamemodify(resolve(local_conf), ':t:r') . '.json'))
```

Comment:
Yeah it's confusing..
Let me try to explain 


If you use Nix (https://nixos.org) to manage your dot files like init.toml
The timestamp of the file is always the same and it's the Unix epoch time (1970)

Because of this the current caching strategy does not work 

In Nix instead every new version of the file changes name because it contains an hash of the file contents in the name 

So between different versions the symlink in
.Spacevim.d/init.toml points to different files 


If we change caching this way it will keep working for existing users with no change but for people like me that use Nix the cache will be busted because the filename has changed.


Does this clarify? I can try to provide a more detailed example if you want 

And yes. I agree this is a limited use case but my argument is that it won't make it worse for anyone else...

Thanks for reading!